docs/UserGuide.md

```diff
@@ -172,98 +168,71 @@ Expected output when a command succeeds:
 Expected output when the command fails:
 * `Invalid command format! edit: Edits the details of the person identified by the index number used in the displayed person list. Existing values will be overwritten by the input values. Parameters: INDEX (must be a positive integer) [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [b/BIRTHDAY] [g/GROUP]…​`
 
-<<<<<<< HEAD
 [Scroll back to Table of Contents](#table-of-contents)
 
-### Listing all persons : `list`
-=======
-### Listing all persons and events: `list_all`
->>>>>>> update_dg
-
-Displays all persons and events stored in FumbleLog
+### Deleting a person : `delete_person`
 
-Format: `list_all`
+Deletes the specified person from FumbleLog.
+When a person is deleted, any [events](#commands-for-events) that the person is assigned to will also be updated, i.e. the person will be unassigned from the event.
 
-Expected output when a command succeeds:
-* Input: `list_all`
-* Output: `Listed all persons and events`
-* You should see a list of all persons and events under the persons and events column.
+Format: `delete_person PERSON_INDEX`
 
-### Listing all events: `list_events`
+* Deletes the person at the specified `PERSON_INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index **must be a positive integer** 1, 2, 3, …​
 
-Displays all events stored in FumbleLog
+Examples:
+* `list_all` followed by `delete_person 2` deletes the 2nd person in the person list.
+* `find_all Betsy` followed by `delete_person 1` deletes the 1st person in the results of the `find` command. i.e Any person named `Betsy` at index `1` will be deleted.
 
-Format: `list_events`
+Acceptable values for each parameter:
+* `PERSON_INDEX`: A positive integer
 
 Expected output when a command succeeds:
-* Input: `list_events`
-* Output: `Listed all events`
-* You should see a list of all events under the Events column.
-
-
-### Listing all persons : `list_persons`
-
-Displays all persons stored in FumbleLog.
+* Input: `delete_person 1`
+* Output: `Deleted Person: Roy Balakrishnan; Phone: 92624417; Email: royb@example.com; Address: Blk 45 Aljunied Street 85, #11-31; groups: [colleagues]`
 
-Format: `list_persons`
+![DeletePerson](images/Deleteperson.png)
 
-Expected output when a command succeeds:
-* Input: `list_persons`
-* Output: `Listed all persons`
-* You should see a list of all persons under the Persons column.
+Expected output when the command fails:
+* `Invalid command format! delete: Deletes the person identified by the index number used in the displayed person list. Parameters: INDEX (must be a positive integer)Example: delete 1`
```

Comment:
Should change delete to delete_person, as well for other invalid outputs for person commands